tensorflow/python/layers/core.py

```diff
@@ -57,6 +57,9 @@ class Dense(base.Layer):
       linear activation.
     use_bias: Boolean, whether the layer uses a bias.
     kernel_initializer: Initializer function for the weight matrix.
+      If `None` (default), weights are initialized using the global default
+      initializer which uses the `glorot_uniform_initializer` defined in
+      `tensorflow.python.ops.init_ops`.
```

Comment:
`tensorflow.python.ops.init_ops` is an internal path, not something that should be publicly referred to. I suggest that instead you refer to the documentation for `get_variable`, which is where this default is configured.